test/presubmit-tests.sh

```diff
@@ -16,83 +16,34 @@
 
 # This script runs the presubmit tests; it is started by prow for each PR.
 
+set -o xtrace
+
+source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/presubmit-tests.sh
+
 function build_tests() {
-  echo "Running build tests"
-  # TODO(aaron-prindle) add build tests
-  echo "TODO(aaron-prindle) add build tests"
+  header "Verifying code structure and contents"
+
+  local result=0
+  subheader "Checking that we can build the code successfully"
+  go build -v ./... || result=1
```

Comment:
Do you also want to #77 in this PR?

Reply:
sure lemme see - as long as `go test` won't return an error when there are no tests yet